Options/WeakAurasOptions.lua

```diff
@@ -7126,13 +7126,21 @@ function WeakAuras.CreateFrame()
         if not(combinedText == "") then
           combinedText = combinedText.."\n\n";
         end
-        combinedText = combinedText.."--"..childId.."\n";
+
+        local formatString
+           = L["-- The following part is the\""]
+             .. childId
+             .. L["\" trigger, do not remove this comment"] .. "\n";
+
+        combinedText = combinedText.. string.format(formatString, childId);
```

Comment:
Ah inedd, that is right, I forgot to remove that, once I realized that I properly don't want to use string.format